### ISSUE TITLE
Auto-generated program.vb throws exception

### ISSUE THREAD

**Issue by singhsarab** [Microsoft/vstest#1907]
## Description
The auto-generated program.vb causes compilation errors if the test code is inside a namespace that starts with Microsoft.

## Steps to reproduce
> Create a new MSTest Net core project for VB
> Rename the Namespace in the UnitTest1.vb to Microsoft.* like:

```
Imports Microsoft.VisualStudio.TestTools.UnitTesting

Namespace Microsoft.FooBar
    <TestClass>
    Public Class UnitTest1
        <TestMethod>
        Sub TestSub()

        End Sub
    End Class
End Namespace
```

## Expected behavior
> The test project should compile without errors

## Actual behavior
> ` <ProjectName>.Program.vb(4,2): error BC30002: Type 'Microsoft.VisualStudio.TestPlatform.TestSDKAutoGeneratedCode' is not defined.`

**Comment by mayankbansal018** [comment]
As discussed with @nguerrera , @tmat going forward solution for test projects would be to use `<HasRuntimeOutput>true</HasRuntimeOutput>` property instead of setting output type to "exe, but <HasRuntimeOutput>, doesn't generate "runtimeconfig.dev.json", which the platform needs to find location of "testhost.dll". 
The issue is currently blocked on https://github.com/Microsoft/msbuild/issues/1310

**Comment by mayankbansal018** [comment]
Duplicate of https://github.com/Microsoft/vstest/issues/792
